subgraph/core/src/entities/Dispute.ts

```
@@ -1,3 +1,4 @@
+import { BigInt, ByteArray, crypto, dataSource, ethereum } from "@graphprotocol/graph-ts";
```

Biome

[error] Do not shadow the global "BigInt" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)